assets/src/application/ui/forms/ee-form/RenderSections.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { RenderSectionsProps } from './types';
+import RenderSection from './RenderSection';
+
+const RenderSections: React.FC<RenderSectionsProps> = ({ sections }) => {
+	return (
+		<div className='sections-wrapper'>
+			{sections.map((section, key) => (
```

Comment:
:)
That's index, not quite `key` :)

Reply:
As per react docs:
`We don’t recommend using indexes for keys if the order of items may change.` from: https://reactjs.org/docs/lists-and-keys.html

Reply:
If order won't change I am ok with using indexes as keys.

Reply:
Where it can change, I have added other values to the keys